src/plays/star-rating/StarRating.js

```diff
@@ -0,0 +1,82 @@
+import PlayHeader from "common/playlists/PlayHeader";
+import { useState } from "react";
+import { FaStar } from "react-icons/fa";
+import "./styles.css";
+
+// WARNING: Do not change the entry componenet name
+function StarRating(props) {
+  // Your Code Start below.
+
+  const [rating, setRating] = useState(null);
+  const [hoverFill, setHoverFill] = useState(null);
+
+  const getReviewLabel = (rating) => {
+    switch (rating) {
+      case 1:
+        return `Very bad ${String.fromCodePoint("0x1F922")}`;
```

Comment:
You can directly copy the emoji here instead of using `String.fromCodePoint`. 
like
![image](https://user-images.githubusercontent.com/7145967/187090608-ae8e04f0-0a52-41e7-949a-2879d7f0e39a.png)


No need to change anything here. I just thought of sharing the info